src/player.rs

```diff
@@ -18,7 +27,9 @@ pub struct PlayerPlugin;
 impl Plugin for PlayerPlugin {
     fn build(&self, app: &mut App) {
         app.add_systems(Startup, spawn_player)
-            .add_systems(Update, player_movement);
+            .add_systems(Update, player_movement)
+            .add_systems(Update, spawn_laser)
+            .add_systems(Update, velocity);
```

Comment:
I think we can make this more concise with `.add_systems(Update, (player_movement, spawn_laser, velocity))`.

Though its not a blocker.